jframework-test/pom.xml

```diff
@@ -47,6 +47,20 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-module-junit4</artifactId>
+            <version>2.0.2</version>
```

Comment:
Why this version of PowerMock?

Reply:
version 2.0.2 ;)
There is `org.mockito:mockito-core:jar:2.23.4:test`  already in their pom